api/identitatem/v1alpha1/strategy_types.go

```diff
@@ -14,16 +15,70 @@ type StrategySpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make generate-clients" to regenerate code after modifying this file
 
-	// Foo is an example field of Strategy. Edit strategy_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// Authentication Realm
+	// +required
+	AuthRealmRef *corev1.LocalObjectReference `json:"authRealmRef,omitempty"`
+
+	// Strategy to use for applying the AuthRealm to the managed clusters
+	// +kubebuilder:validation:Enum=backplane;grc;hive
+	// +required
+	StrategyType StrategyType `json:"type"`
+
+	//default to enforce but allow user to change, OPTIONAL - only used when type is grc
+	// +kubebuilder:validation:Enum=enforce;inform
+	RemediationType RemediationActionType `json:"remediationAction,omitempty"`
```

Comment:
Can't you leverage the ActionType already defined in the GRC project instead of creating your own RemediationActionType?

Reply:
No longer part of the CRD after a redesign...